packages/docs/babel.config.js

```diff
@@ -12,6 +12,7 @@ const isProd = process.env.NODE_ENV === 'production';
 
 const options = {
   dev: !isProd,
+  runtimeInjection: !isProd,
```

Comment:
Please can we revert this? We're looking remove runtime injection altogether eventually, and it's false by default now